### PR TITLE
Update product grid

### PR DESCRIPTION
**Pull request by thomasdigby**

### CODE REVIEW

File: resources/views/category.twig [hunk 1]
```diff
@@ -1,37 +1,23 @@
...
+	      <li class="product_collection-link {% if category.name|lower == product_collection_category.name|lower %}product_collection-link-active{% endif %}">
+          <a href="{{site.url}}/shop/{{product_collection.slug}}/{{category.name|lower}}" class="product_collection-link-category">{{category.name}}</a>
+        </li>
+	    {% endfor %}
+		</ul>
+  </div>
 
-    {% endfor %}
+  {% for sub_category in product_collection_category.sub_categories %}
+    {% include "@AgreableProductPlugin/product-collection/template.twig" with {
```
Comment (ecoad) [inline resources/views/category.twig]: Should this template be named something that reflects that it is a sub category template? `@AgreableProductPlugin/product-collection/template.twig`, judging by that filepath I'd read it that it was the main page template.

Comment (thomasdigby) [reply resources/views/category.twig]: Yes agreed, it should be product-list to match the style/component.

**Comment by KimberleyCook** [comment]
screen shot 2016-08-24 at 17 25 46

This title view is slightly broken now, also can we make sure the sub headers 'YOU MAY ALSO LOVE' are bold again please

**Comment by thomasdigby** [comment]
That's a global reset applying to heading font-weights. I'll remove it for now.

**Comment by KimberleyCook** [comment]
👍

**Pull request merged**